metrics/run/collect_metrics.go

```diff
@@ -211,7 +212,14 @@ func main() {
 		*inputGcsBucket)
 
 	readStartTime := time.Now()
-	runs := base.FetchLatestRuns(*wptdHost)
+	var runs []metrics.TestRunLegacy
+	runsWithLabels := base.FetchLatestRuns(*wptdHost)
+	if serialized, err := json.Marshal(runsWithLabels); err != nil {
+		log.Fatal(err)
+	} else if err = json.Unmarshal(serialized, &runs); err != nil {
+		log.Fatal(err)
+	}
```

Comment:
This is another JSON round trip just to copy the fields, right? Perhaps we should have a helper function in `models.go`.

Reply:
Done.